packages/mobile/views/dashboard/tabs/ActivityTab.svelte

```diff
@@ -1,20 +1,30 @@
 <script lang="ts">
-    import { isStrongholdUnlocked } from '@core/profile-manager'
     import { Activity } from '@core/wallet'
     import { ActivityList } from '../../../../mobile/components'
-    import { ActivityAction } from '../../../lib/contexts/dashboard'
-    import { activityRouter } from '../../../lib/routers'
+    import { DrawerId, openDrawer } from '../../../lib/auxiliary/drawer'
+    import { handleClaimActivity, handleRejectActivity } from '../../../lib/contexts/dashboard'
 
     function onTileClick(activity: Activity): void {
-        $activityRouter?.next({ activity })
+        // TODO: when token drawer is refactor, implement this
+        // if (activity.type === ActivityType.Basic) {
+        //     const asset = getAssetById(activity.assetId)
+        //     // TODO: open token drawer to verify/skip token
+        // } else {
+        //     openDrawer(DrawerId.SelectedActivity, {
+        //         activity,
+        //     })
+        // }
```

Comment:
Hmm why would this logic be a part of the tab component?

Reply:
this logic in not in a component per se (though every .svelte files is a component ), this file is the full activity tab view, like a view parent that takes care of all the "routing" logic, in this case, the drawer logic. This logic is then passed to the `ActivityList` component, I dont see why is wrong having this logic here, what would you suggest instead?

Reply:
the logic is uncommented and implemented on its corresponding PR already